### ISSUE TITLE
Lua errors on login with 6.0.0 

### ISSUE THREAD

**Issue by tehKD** [AeroScripts/QuestieDev#2118]
I updated to the latest version (6.0.0)
On every log in im getting these errors:
```
1x Questie\Libs\ZoneHelper\ZoneHelper.lua:32: attempt to index local 'helper' (a nil value)
Questie\Libs\ZoneHelper\ZoneHelper.lua:32: in main chunk

Locals:
AceEvent = <table> {
 embeds = <table> {
 }
 messages = <table> {
 }
 RegisterEvent = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90
 UnregisterAllEvents = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:174
 OnEmbedDisable = <function> defined @DataStore\libs\AceEvent-3.0\AceEvent-3.0.lua:112
 Embed = <function> defined @DataStore\libs\AceEvent-3.0\AceEvent-3.0.lua:99
 SendMessage = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:59
 events = <table> {
 }
 UnregisterAllMessages = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:174
 UnregisterMessage = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:153
 frame = AceEvent30Frame {
 }
 UnregisterEvent = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:153
 RegisterMessage = <function> defined @DataStore\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:90
}
helper = nil
zoneIDToUiMapID = <table> {
}
listeners = <table> {
}
previousSubZone = "Altar of Storms"
previousZone = "Burning Steppes"
updateListeners = <function> defined @Questie\Libs\ZoneHelper\ZoneHelper.lua:13
(*temporary) = <function> defined @Questie\Libs\ZoneHelper\ZoneHelper.lua:32
(*temporary) = 1
(*temporary) = "%d+"
(*temporary) = "1"
(*temporary) = "attempt to index local 'helper' (a nil value)"
```

```

1x ...tterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:22: attempt to call method 'UpdateHiddenNotes' (a nil value)
...tterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:22: in function <...tterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:18>
SharedXML\C_TimerAugment.lua:16: in function <SharedXML\C_TimerAugment.lua:14>

Locals:
questie = <table> {
 DB = <table> {
 }
 Quest = <table> {
 }
}
(*temporary) = nil
(*temporary) = <table> {
 GetAllQuestIds = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:450
 AcceptQuest = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:319
 HideQuest = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:261
 ToggleNotes = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:61
 PopulateQuestLogInfo = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:896
 AbandonedQuest = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:370
 SmoothReset = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:221
 ShouldQuestShowObjectives = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:497
 PopulateObjectiveNotes = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:855
 private = <table> {
 }
 CalculateAvailableQuests = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:1395
 GetRawLeaderBoardDetails = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:272
 UpdateQuest = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:421
 UnhideQuest = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:266
 GetAllQuestIdsNoObjectives = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:478
 DrawAllAvailableQuests = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:1292
 AddAllNotes = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:188
 PopulateObjective = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:616
 GetAllQuestObjectives = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:917
 availableQuests = <table> {
 }
 UpdateObjectiveNotes = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:509
 AddFinisher = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:521
 CalculateAndDrawAvailableQuestsIterative = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:1311
 CompleteQuest = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:343
 Initialize = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:51
 GetAllLeaderBoardDetails = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:1147
 ForceToMap = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:586
 ClearAllNotes = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:148
 Reset = <function> defined @Questie\Modules\Quest\QuestieQuest.lua:203
}
(*temporary) = "attempt to call method 'UpdateHiddenNotes' (a nil value)"
helper = <table> {
 GetAddons = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:44
 SetAutoHideQuestHelperIcons = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:111
 SetIconsVisibility = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:143
 GetQuestie = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:54
 autoHideIcons = false
 GetActiveAddons = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:85
 IsSupported = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:97
 GetDistanceToClosestObjective = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:173
 RefreshIconsVisibilityForQuests = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:123
 GetAddonNames = <function> defined @ButterQuestTracker\LibQuestHelpers\LibQuestHelpers.lua:75
}
```

**Comment by drejjmit** [comment]
Looks like a compability issue with ButterQuestTracker. Questie should still work but the new Questie Tracker won't. We'll look into it.

**Comment by nick-woodward** [comment]
Hey @tehKD, I just released [v1.9.25](https://www.curseforge.com/wow/addons/butter-quest-tracker/files/3000787) of ButterQuestTracker which should resolve any compatibility issues with Questie v6! 🎉 

If you run into any further issues, feel free to open an issue on our repository!

https://github.com/butter-cookie-kitkat/ButterQuestTracker

**Comment by BreakBB** [comment]
Hey @nick-woodward Thanks for fixing this issue on your side 🙏
